test/helpers/e2e.go

```diff
@@ -184,14 +197,14 @@ func L2Setup(t *testing.T, cfg *EnvironmentConfig) *L2Environment {
 	t.Helper()
 
 	l2Client, authL2, gerL2Addr, gerL2Contract,
-		bridgeL2Addr, bridgeL2Contract := newSimulatedEVML2SovereignChain(t)
+		bridgeL2Addr, bridgeL2Contract, aggOracleCommitteeAddr, aggOracleCommitteeContract := newSimulatedEVML2SovereignChain(t, cfg.AggoraclecommitteeConfig)
```

Comment:
Linter complains about line being too long.
```suggestion
	l2Client, authL2, gerL2Addr, gerL2Contract,
		bridgeL2Addr, bridgeL2Contract,
		aggOracleCommitteeAddr, aggOracleCommitteeContract := newSimulatedEVML2SovereignChain(t, cfg.AggoraclecommitteeConfig)
```

Reply:
done